corehq/ex-submodules/phonelog/migrations/0005_add_forceclose_entry_20160408_1530.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('phonelog', '0004_merge'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ForceCloseEntry',
+            fields=[
+                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
+                ('domain', models.CharField(max_length=100, db_index=True)),
```

Comment:
Is that basically a more expensive index which would allow for searches and partial matching on this field (which we obviously don't need)?  I'm not sure I understand what you mean by "the name should be standard across all DB'ds" - do you know of another migration which does what you describe?  I looked at several other SQL models in HQ with domain fields, and they all seem to be treated the same (both in `models.py` and in the migrations).  We might need to update all of those too.